pkg/appendable/csv_handler.go

```diff
@@ -74,29 +74,18 @@ func (c CSVHandler) Synchronize(f *IndexFile) error {
 }
 
 func fieldRankCsvField(fieldValue any) int {
-	fieldStr, ok := fieldValue.(string)
```

Comment:
no need to convert `givenType` -> `String` -> `givenType`.

We can just directly case through the `givenTypes`